.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
```

Comment:
Rename this to something like « Linux CI ». It is not excluded that there will be a macOS build in the future, most likely in another `.yml` file.

This will appear in PRs as « GitHubActionName / JobName ».